components/productions/create/event.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <b-card class="card-event">
+    <section class="header">
+      <div>
+        <p>{{ type }}</p>
+        <h2>{{ title }}</h2>
+        <p>{{ period }}</p>
+      </div>
+      <img :src="imageUrl" :alt="title" class="image" />
+    </section>
+    <b-card-text>{{ description }}</b-card-text>
+  </b-card>
+</template>
+
+<script>
+  import { format } from 'date-fns';
+
+  export default {
+    props: {
+      type: {
+        type: String,
+        default: '',
+      },
+      title: {
+        type: String,
+        default: '',
+      },
+      startDate: {
+        type: String,
+        default: '',
+      },
+      endDate: {
+        type: String,
+        default: 'new Date()',
+      },
+      description: {
+        type: String,
+        default: '',
+      },
+      imageUrl: {
+        type: String,
+        default: '',
+      },
+    },
+    computed: {
+      period() {
+        const parsedStart = this.parseDate(this.startDate);
+        const parsedEnd = this.parseDate(this.endDate);
+        if (parsedStart === parsedEnd) {
+          return parsedStart;
+        }
+        return `${parsedStart} - ${parsedEnd}`;
```

Comment:
It would probably be better to use the calendar summary here.
Start and end date can be very vague for events that run multiple times over a longer period of time.
@corneelwille what do you think?

You can fetch it by requesting https://<api>/events/{id}/calsum?format={sm|md|lg}&langCode=nl_BE

For example: https://udb-silex-acc.uitdatabank.be/events/5781ac48-6a92-4ee5-bb67-c0187349b3b4/calsum?format=lg&langCode=nl_BE